app/src/main/java/com/github/se/cyrcle/CyrcleNavHost.kt

```diff
@@ -62,7 +61,12 @@ fun CyrcleNavHost(
         route = Route.MAP,
     ) {
       composable(Screen.MAP) {
-        MapScreen(navigationActions, parkingViewModel, userViewModel, mapViewModel)
+        MapScreen(
+            navigationActions,
+            parkingViewModel,
+            userViewModel,
+            mapViewModel,
+            permissionGranted = permissionGranted)
```

Comment:
[Non-blocking] Why specify only this argument ?

Reply:
because we have the zoom state before in the list of argument which would push me to either add `zoom = defaultzoom` or do that

Reply:
Missed that, sorry for the comment